opentelemetry-appender-log/src/lib.rs

```diff
@@ -239,7 +239,7 @@ mod any_value {
     pub(crate) fn serialize(value: log::kv::Value) -> Option<AnyValue> {
         struct ValueVisitor(Option<AnyValue>);
 
-        impl<'kvs> log::kv::VisitValue<'kvs> for ValueVisitor {
+        impl log::kv::VisitValue<'_> for ValueVisitor {
```

Comment:
... and this one.
I can remove these, but we should probably start enforcing this in CI anyway, I reckon :D